app/styles/layouts/auth.css

```diff
@@ -17,6 +17,10 @@
     margin-bottom: 1.5rem;
 }
 
+.gh-signin .form-group.error {
```

Comment:
Has the behaviour here changed? Last time I looked the field should be highlighted red with the error message displayed underneath the form (same place server-related errors are displayed). There was some discussion about that somewhere but it was a long time ago, I can see if I can find it if needed.